*** 50unattended-upgrades.orig	Fri Feb 28 23:18:08 2020
--- 50unattended-upgrades	Fri Feb 28 23:18:52 2020
***************
*** 14,18 ****
  	"${distro_id}ESM:${distro_codename}-security";
! //	"${distro_id}:${distro_codename}-updates";
  //	"${distro_id}:${distro_codename}-proposed";
! //	"${distro_id}:${distro_codename}-backports";
  };
--- 14,18 ----
  	"${distro_id}ESM:${distro_codename}-security";
! 	"${distro_id}:${distro_codename}-updates";
  //	"${distro_id}:${distro_codename}-proposed";
! 	"${distro_id}:${distro_codename}-backports";
  };
